<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>odunc_verme_islemleri</class>
 <widget class="QDialog" name="odunc_verme_islemleri">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_Odunc_Kitap_Teslim_Et">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>260</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödüncü Ver</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Teslim_Edilen_Kitaplar_Listesi">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>70</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_Teslim_Edilen_Kitaplar_Listesi">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>40</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Teslim Edilen Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_Odunc_Alinan_Kitaplar_Listesi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>291</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Uye_No">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>110</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Odunc_Alinan_Kitaplar_Listesi">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Alınan Kitaplar Listesi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Uye_No">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Üye No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Odunc_Teslim_Etme_Islemleri">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>ÖDÜNÇ TESLİM ETME İŞLEMLERİ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Kitap_No">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>160</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Kitap_No">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kitap No:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Odunc_Verme_Tarihi">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ödünç Verme Tarihi:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit_Odunc_Verme_Tarihi">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>210</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>9</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2024</year>
     <month>6</month>
     <day>10</day>
    </datetime>
   </property>
   <property name="minimumDateTime">
    <datetime>
     <hour>8</hour>
     <minute>0</minute>
     <second>0</second>
     <year>1999</year>
     <month>12</month>
     <day>31</day>
    </datetime>
   </property>
   <property name="date">
    <date>
     <year>2024</year>
     <month>6</month>
     <day>10</day>
    </date>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
